src/org/usfirst/frc/team2706/robot/commands/autonomous/plays/ExampleParallelPlay.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc.team2706.robot.commands.autonomous.plays;
+
+import org.usfirst.frc.team2706.robot.commands.autonomous.movements.StraightDriveWithTime;
+import org.usfirst.frc.team2706.robot.commands.camera.SearchForTarget;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ExampleParallelPlay extends CommandGroup {
+
+	public ExampleParallelPlay() {
+		//Uses camera to search for a target for 5 seconds at the same time as driving forward
```

Comment:
To keep it consistent with the rest of the code, maybe add a space after the `//`